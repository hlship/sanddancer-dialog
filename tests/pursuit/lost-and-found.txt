exit
(setup pursuit)
s
n
n
w
s
s
s
s
s
s
s
s
s
nw
sw
nw
nw
