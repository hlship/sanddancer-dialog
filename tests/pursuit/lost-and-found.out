

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> exit
The door creaks open with difficulty, rusted joints straining. You get out of
the pickup truck.

> (setup pursuit)
Pursuit should occur soon.
Query succeeded: (setup pursuit)
> n
You walk north.

Crumbling Concrete
This concrete building must've been some sort of utility structure for the huge
electrical tower to the north, now abandoned. Tendrils of desert sand drift
against a boarded-up door to the northeast leading inside; you could also walk
around to the east or head south back to your truck.

Out of the corner of your eye you spot a glimpse of something moving in the
darkness, off in the desert to the west.

> w
You head off in the direction of movement, scanning with your flashlight over
the skittering shadows.

You walk west for a long time, thoughts turning over like the sand under your
sneakers. You think about your grandma's stories. You think about Ocean and how
pissed she's gonna be that you're not home. Like you tell her not to wait up for
you but she always does anyway.

Some wind kicks up and chucks sand in your face. You blink it out and kinda
realize while you're doing it that you haven't really been paying attention to
where you've been going. Another gust whines through the sagebrush and you
realize with a kind of heavy feeling that with all the dust this wind is kicking
up you can't see a damn thing. You can't see your pickup, you can't see the
electrical tower. Not even your damn footprints. You're lost. Well, shit.

Open Desert
The dust storm rages around you, and you plod wearily through the sand.

Nearby: a low bluff, a dead cow, and a looming cactus. Nothing to write home
about.

Yeah, you're sure you saw it--something's moving to the west.

> w
You chase after the shadow, faster and faster, breath misting in the cold air.

Open Desert
The desert stretches around you in all directions, and you wish you could see
more than twenty feet in front of you.

All you can make out in the darkness are a stand of cactus, a low bluff, and a
dry streambed.

Hey, there it is again--a darting shadow to the west.

> s
You lose sight of whatever it was.

You walk south.

Open Desert
The dust storm rages around you, and you shiver in the chill night air.

Half-lost in the shadows, you see a stand of cactus, a looming cactus, and a low
bluff and that's about it.

> w
You walk west.

Open Desert
The desert stretches around you in all directions, and you wish you could see
more than twenty feet in front of you.

You can sort of see some dry bones, a looming cactus, and a dead cow.

> w
You walk west.

Open Desert
Your tracks are swallowed up in moments by the billowing sand, and god damn it's
dark.

Half-lost in the shadows, you see some clumps of sagebrush, a looming cactus,
and a dry streambed and that's about it.

You see that weird movement again, out in the shadows to the west.

> w
You head off in the direction of movement, scanning with your flashlight over
the skittering shadows.

Open Desert
Clouds of dust swirl through the air, and every inch of sand looks just like
every other.

Nearby: a looming cactus, a dry streambed, and a stand of cactus. Nothing to
write home about.

Yeah, you're sure you saw it--something's moving to the west.

> w
You chase after the shadow, faster and faster, breath misting in the cold air.

Open Desert
The dust storm rages around you, and you plod wearily through the sand.

All you can make out in the darkness are a dry streambed, some clumps of
sagebrush, and a stand of cactus.

Hey, there it is again--a darting shadow to the west.

> w
Some crazy thrill comes over you. You feel like a hunter. You can almost smell
the thing you're chasing in the air. You can smell it, fur and terror. You race
after it.

Open Desert
Clouds of dust swirl through the air, and god damn it's dark.

Half-lost in the shadows, you see a dead cow, a stand of cactus, and a looming
cactus and that's about it.

It's gone now... no, wait--something's moving to the west.

> w
Dodging after the fleeing, skittering shadow, you're almost caught up to it when
your foot trips on a rock and you stumble to the ground. But the ground doesn't
stop your fall, it collapses, and you're tumbling in a shower of dirt clods down
a hole, rolling down some crazy underground tunnel clutching at mud and roots,
till finally you slide into some damp, fetid place, and come up a stop against
something soft, huge, warm. And furry.

Burrow
Roots push through the earthen roof, casting weird shadows in the beam of your
flashlight. It's round and small and underground in here and filled with hot
sweat and animal stench. But mostly filled with something big and alive.

It's huge, covered in sweaty fur stained with mud, and a huge eye stares out you
above long black whiskers. You can feel its breath on your face and its
heartbeat thudding through the floor and you cringe back in terror against the
wall. It'sâ€¦ well.

It's a rabbit.

The rabbit breaths and stares. You wonder if it's going to start talking like in
some crap disney movie and then jesus christ it does...

"So, what'd you expect from an animal guardian?" it says, but like not with its
lips, somewhere in your head instead. "Were you thinking big, nasty, sharp,
pointy teeth? Waistcoat and pocket watch? Nah. Tricks are for kids. What's up
Knock?" Its whiskers twitch.

You could ask who he is or just, you know, go insane.

> 
